lib/api/objectPut.js

```diff
@@ -45,10 +44,19 @@ function _storeIt(bucketName, objectKey, objMD, authInfo, canonicalID,
         authInfo,
         metaHeaders,
         size,
-        contentType,
+        contentType: request.headers['content-type'],
+        cacheControl: request.headers['cache-control'],
+        contentDisposition: request.headers['content-disposition'],
+        expires: request.headers.expires,
         headers,
         log,
     };
+    // Remove V4 streaming value 'aws-chunked' if present in content-encoding to
```

Comment:
Having the comment in one place is good.  However, could you put the utility function in this file:
https://github.com/scality/S3/tree/master/lib/api/apiUtils/object